tests/simple.rs

```diff
@@ -30,9 +30,9 @@ fn simple() {
     };
     let mut manager = PeerNetManager::new(config);
     manager
-        .start_listener(TransportType::Tcp, "127.0.0.1:8080".parse().unwrap())
+        .start_listener(TransportType::Tcp, "127.0.0.1:64850".parse().unwrap())
         .unwrap();
-    //manager.start_listener(TransportType::Quic, "127.0.0.1:8081".parse().unwrap()).unwrap();
+    //manager.start_listener(TransportType::Quic, "127.0.0.1:64850".parse().unwrap()).unwrap();
     let clients = create_clients(11);
```

Comment:
You need to pass the port to the client helper creation function if you change the listener port

Reply:
I added it to the `create_clients` function also